frontend/src/components/Calendar/CalendarDay.tsx

```diff
@@ -48,13 +48,13 @@ const useStyles = makeStyles((theme: Theme) => ({
 
 const CalendarDay: React.VFC<Props> = (props) => {
```

Comment:
```suggestion
const CalendarDay: React.VFC<Props> = ({ onClick, value, isHidden }) => {
```

Reply:
nope, props is used for useStyles in line 50